packages/core/rendering/core-rendering-server-internal/src/bootstrap/render_template.ts

```diff
@@ -68,15 +68,38 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       // make subsequent calls to failure() noop
       failure = function () {};
 
-      var err = document.createElement('h1');
-      err.style['color'] = 'white';
-      err.style['font-family'] = 'monospace';
-      err.style['text-align'] = 'center';
-      err.style['background'] = '#F44336';
-      err.style['padding'] = '25px';
-      err.innerText = document.querySelector('[data-error-message]').dataset.errorMessage;
-
-      document.body.innerHTML = err.outerHTML;
+      var errorTitle =  document.querySelector('[data-error-message-title]').dataset.errorMessageTitle;
+      var errorText =  document.querySelector('[data-error-message-text]').dataset.errorMessageText;
+      var errorReload =  document.querySelector('[data-error-message-reload]').dataset.errorMessageReload;
+
+      var err = document.createElement('div');
+      err.style.textAlign = 'center';
+      err.style.padding = '120px 20px';
+      err.style.fontFamily = 'Inter, BlinkMacSystemFont, Helvetica, Arial, sans-serif';
+
+      var errorTitleEl = document.createElement('h1');
+      errorTitleEl.innerText = errorTitle;
+      errorTitleEl.style.margin = '20px';
+
+      var errorTextEl = document.createElement('p');
+      errorTextEl.innerText = errorText;
+      errorTextEl.style.margin = '20px';
+
+      var errorReloadEl = document.createElement('button');
+      errorReloadEl.innerText = errorReload;
+      errorReloadEl.onclick = function () {
+        location.reload();
+      };
+      errorReloadEl.setAttribute('style',
+       'cursor: pointer; padding-inline: 12px; block-size: 40px; font-size: 1rem; line-height: 1.4286rem; border-radius: 6px; min-inline-size: 112px; color: rgb(255, 255, 255); background-color: rgb(0, 119, 204); outline-color: rgb(0, 0, 0); border:none'
+      );
```

Comment:
We're in 2008. jQuery just got out of beta, and developers are still forced to have their "code" work on IE5.5 and IE6. (jocking, doc.querySelector wasn't a thing for IE6)